js/src/components/lineage/GraphNode.tsx

```diff
@@ -106,20 +106,23 @@ export function GraphNode({ data }: GraphNodeProps) {
   const { selectNodeMulti, selectMode } = useLineageViewContext();
 
   // text color, icon
-
   let color = "gray.400";
+  let backgroundColor = "gray.100";
   let iconChangeStatus: any;
   let borderStyle = "solid";
   if (changeStatus) {
     iconChangeStatus = getIconForChangeStatus(changeStatus).icon;
     color = getIconForChangeStatus(changeStatus).color;
+    backgroundColor = getIconForChangeStatus(changeStatus).backgroundColor;
```

Comment:
would this be better instead of calling the function three times?
```
  const iconForChangStatus = getIconForChangeStatus(changeStatus);
  iconChangeStatus = iconForChangStatus.icon;
  color = iconForChangStatus.color;
  backgroundColor = iconForChangStatus.backgroundColor;
```

Reply:
Fixed